### ISSUE TITLE
It does not work on Linux wps

### ISSUE THREAD

**Issue by victory1056** [tankwyn/WPS-Zotero#14]
When I ran the program install.py on the platform of deepin linux，it does not work on my wps.Zotero did not appear on the custom tab.Could you tell us how to use it on the linux wps.

**Comment by tankwyn** [comment]
Last time I checked it works fine on my Linux laptop. What is the output of install.py? What are your WPS, Zotero, WPS-Zotero and Python versions?

**Comment by victory1056** [comment]
> Last time I checked it works fine on my Linux laptop. What is the output of install.py? What are your WPS, Zotero, WPS-Zotero and Python versions?
This is the output when I executed the program of install.py
![Screenshot_select-area_20231221110334](https://github.com/tankwyn/WPS-Zotero/assets/55820017/cace51c5-7818-41cb-9c99-f6fcf453f3b3)
My WPS verison and Zotero version are as follows.
![Screenshot_select-area_20231221110551](https://github.com/tankwyn/WPS-Zotero/assets/55820017/63baf232-9d00-41d2-917e-181ed3ae823d)
![Screenshot_select-area_20231221110724](https://github.com/tankwyn/WPS-Zotero/assets/55820017/3b3918c6-8020-4bd0-b60e-2d4cb1b20d84)
![Screenshot_select-area_20231221111222](https://github.com/tankwyn/WPS-Zotero/assets/55820017/168799f5-d4ba-4836-9600-e9d0990d1fcf)
I do not know the version of WPS-Zotero，I guess it may be the program you shared on the github. Is that right?
![Screenshot_select-area_20231221111714](https://github.com/tankwyn/WPS-Zotero/assets/55820017/fb1b2dc0-dc6a-4906-bb48-551aed94787d)

**Comment by jiandwei** [comment]
I also have similar problem, when I open WPS and Zotero, it just works only once. Then when I close it and restart them again. It will report such a problem. But,actually, Zotero and WPS are working.
![image](https://github.com/tankwyn/WPS-Zotero/assets/148776856/2f4a8606-8736-49e8-b5a5-d877fcfd67ce)

**Comment by tankwyn** [comment]
@victory1056 
You shouldn't run the install script as root. The whole thing doesn't require root privilege and installs to `/home`.

**Comment by tankwyn** [comment]
@jiandwei
Please open another issue and respond in the new issue.
Please check whether proxy.py is still running when this error occurs and upload ~/.wps-zotero-proxy.log.

**Comment by victory1056** [comment]
> @victory1056 You shouldn't run the install script as root. The whole thing doesn't require root privilege and installs to `/home`.

Thanks for your help.Now I can see zotero appear on the wps.
![image](https://github.com/tankwyn/WPS-Zotero/assets/55820017/1fa9ebe4-bbaf-439c-b6ba-42b62f5c4fb5)
However I met another quetion.When I insert citition in the wps,it occured a error.The quetison  is as follows.I don not know how to solve it.
![dingtalkgov_qt_clipbord_pic_2](https://github.com/tankwyn/WPS-Zotero/assets/55820017/09c60784-1997-4a34-a8e4-ce441e044299)

**Comment by tankwyn** [comment]
@victory1056 This indicates the citations in your document were created by another word processor like Libreoffice Writer. Note that WPS-Zotero is compatible only to the MS Word integration. You should open this document in the word processor that created it and use the export button to export the document. Then close the document and open it in WPS and click the 'Refresh' button to import it.

**Comment by victory1056** [comment]
> @victory1056 This indicates the citations in your document were created by another word processor like Libreoffice Writer. Note that WPS-Zotero is compatible only to the MS Word integration. You should open this document in the word processor that created it and use the export button to export the document. Then close the document and open it in WPS and click the 'Refresh' button to import it.

Thanks for your help  sincerely.I had solved the problem.I creat a new document that  can sovle the  problem.The best wish to you.
